src/reference-designs/tko-on-vsphere-with-tanzu.md

```diff
@@ -310,10 +310,13 @@ The initial configuration and setup of the platform does not require any externa
 
 When vSphere with Tanzu is deployed on VDS networking, you can deploy an external container registry (Harbor) for Tanzu Kubernetes clusters.
 
-There are two main supported installation methods for Harbor:
+There are three officially documented installation methods for Harbor:
```

Comment:
The following installation methods are supported for Harbor:

Reply:
This should be addressed in the latest push